Northstar.CustomServers/mod/scripts/vscripts/evac/_evac.gnut

```diff
@@ -391,7 +393,7 @@ void function AddPlayerToEvacDropship( entity dropship, entity player )
 	EmitSoundOnEntityOnlyToPlayer( player, player, SHIFTER_START_SOUND_3P )
 	// should play SHIFTER_START_SOUND_1P when they actually arrive in the ship i think, unsure how this is supposed to be done
 	PlayPhaseShiftDisappearFX( player )
-	waitthread FirstPersonSequence( fp, player, dropship )
+	FirstPersonSequence( fp, player, dropship )
```

Comment:
FirstPersonSequence calls WaittillAnimDone,  shouldn't it be threaded?

Reply:
@BobTheBob9

Reply:
> FirstPersonSequence calls WaittillAnimDone, shouldn't it be threaded?

this is already within a thread so doesn't matter, either way we block execution of the thread